package.json

```diff
@@ -43,7 +43,7 @@
     "bootstrap-sass": "^3.3.7",
     "broccoli-asset-rev": "^2.7.0",
     "ember-auto-import": "^2.4.0",
-    "ember-bootstrap": "4.9.0",
```

Comment:
do we need to update `ember-bootstrap` ?

Reply:
Not necessarily, but upgrading eliminates a bunch of deprecation noise and is already forced for the ember-release tests